UnityProject/Assets/Scripts/Managers/CentComm.cs

```diff
@@ -83,6 +83,10 @@ public void UpdateStatusDisplay(StatusDisplayChannel channel, string text)
 	private string AntagInitialUpdate =
 		"Enemy communication intercepted. Security level elevated.";
 
+	public static string BioHazardReportTemplate =
```

Comment:
Codacy found an issue: [Change the visibility of 'BioHazardReportTemplate' or make it 'const' or 'readonly'.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6357029)